manifests/d/Dapr/CLI/1.9.1/Dapr.CLI.locale.en-US.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Dapr.CLI
+PackageVersion: 1.9.1
+PackageLocale: en-US
+Publisher: Dapr
+PublisherUrl: https://dapr.io
+PackageName: CLI
+PackageUrl: https://github.com/dapr/cli
+License: Apache 2.0
+LicenseUrl: https://github.com/dapr/cli/blob/master/LICENSE

```

Comment:
Can the license point to the v1.9.1 tag? It is unlikely we will change it but still nice to make it a immutable URL.

Reply:
Approving for now. Please, change in the upcoming releases.

Reply:
I used the LICENSE in master so that it could be kept constant across releases. We plan to use [wingetcreate update](https://github.com/microsoft/winget-create/blob/main/doc/update.md) to update the manifests in workflow itself at every release (https://github.com/dapr/cli/pull/1068) , which seemingly doesn't support updating `LicenseUrl` automatically (I don't see any such sub-flag in [wingetcreate update](https://github.com/microsoft/winget-create/blob/main/doc/update.md) ). Thereby this would require manual update in every release, which shouldn't be done ideally.

If changing of License gets to be an issue, we can also remove `LicenseUrl` completely. It's not a required field. 
cc @ryfu-msft